src/components/ProjectModal.vue

```diff
@@ -169,10 +184,21 @@ export default defineComponent({
     this.loadProject();
   },
   methods: {
+    async shareProject() {
+      await Share.share({
+        title: this.title,
+        text: 'Check out this Scratch project I found with Itchy!',
```

Comment:
IMO, this text is pretty unnecessary. I personnally dislike when apps add text to shared links.

Reply:
Agreed.  I'll fix that this afternoon.